src/ts/codeBlocks.ts

```diff
@@ -11,14 +11,16 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+import { listen , copyToClipboard , getById} from "./utils";
+import { button , ariaLabel , mouseenter , mouseleave , click , active} from "./constants";
+import { readLocalStorage } from "./themes_init";
```

Comment:
review top tip: I don't know anything about Typescript but it would be worth providing a little context as to why these need to be added